sdcm/cluster.py

```diff
@@ -4565,6 +4565,16 @@ def node_setup(self, node, verbose=False, db_node_address=None, **kwargs):  # py
         if self.params.get('client_encrypt'):
             node.config_client_encrypt()
 
+        # Install java-8 for workaround hdrhistogram
```

Comment:
Can you add more information as to why this can't work with newer java ? defect number of the issue.

We should strive to fix it, and not work around it.

I.e. is the c-s from scylla docker image also broken like this ? (Since we want to switch to using that one)

Reply:
There is/was an issue with our c-s and newer JDK.
This is temporary workaround, I think it should be fixed by now. isn't it @aleksbykov ?

Reply:
check with the latest scylla docker image. cassandra stress with -log hdrfile=file.hdr is still failed to run
i add new docker image which is use java8, once we witched to cs in docker for performance tests this part could be removed. but it required if cs runs as process on instance